src/control/run_go_tests.sh

```diff
@@ -117,50 +117,6 @@ function setup_environment()
 	fi
 }
 
-function emit_junit_failure()
```

Comment:
What's the motivation for removing this function? If it's not currently needed, then we could adjust the logic so that it's not called for the current CI environment. I'm hesitant to just toss it altogether though, as this precise format was found to work with Jenkins' JUnit parser via trial and error.